File: Selecao I/guarda.cpp
/*@guarda: Retornando problemas primeiro*/

#include <iostream>
#include <locale>
using namespace std;

int main(){
    setlocale(LC_ALL, "Portuguese");
    int wifi, login, admin;
    
    cin >> wifi;
    cin >> login;
    cin >> admin;
    
    if(wifi == 0) cout << "you must connect to wifi" << endl;
    else if(login == 0) cout << "you need to login first" << endl;
    else if(admin == 0) cout << "you must to login as admin" << endl;
    else cout << "done" << endl;
    return 0;
}

File: src/Selecao_I/Guarda.java
/**
 * Questão: @guarda: Retornando problemas primeiro
 *
 * @author UxieGu1
 */
package Selecao_I;
import java.util.Scanner;

public class Guarda {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        int wifi = scan.nextInt();
        int login = scan.nextInt();
        int admin = scan.nextInt();

        if(wifi == 0) {
            System.out.println("you must connect to wifi");
        } else if(login == 0) {
            System.out.println("you need to login first");
        } else if(admin == 0) {
            System.out.println("you must to login as admin");
        } else {
            System.out.println("done");
        }

        scan.close();
    }
}
